introduction.md

```diff
@@ -1 +1,9 @@
 Greetings, Vipers! Love the JavaScript path!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?